src/price_data.py

```diff
@@ -516,6 +522,70 @@ def get_price(
 
         return price
 
+    def get_missing_price_operations(
```

Comment:
Would it be sufficient to drop all elements of `operations` that are coin lend, coin lend end, staking, staking end etc. at the beginning of the function? Theoretically this could be different for each country, so it might make sense to differentiate here before dropping the elements.